orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -183,7 +183,7 @@
              </param>
              <param>
                  <name>{{ aclsauthz_prefix ~ "resourcemanager.acl" }}</name>
-                 <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
```

Comment:
This, and the subsequent changes - it'll need the field which set the value to an OR for the specific acl.

Reply:
Added the param to set the value to OR for the `resourcemanager.acl`